scripts/happy

```diff
@@ -253,7 +257,10 @@ class Stack:
             sensitive=False,
         )
         for k, v in self.meta.parameters.items():
-            self.workspace.set_var(k, str(v), "", sensitive=False)
+            if k == "image_tags":
+                self.workspace.set_var(k, json.dumps(v), "", sensitive=False)
+            else:
+                self.workspace.set_var(k, str(v), "", sensitive=False)
```

Comment:
This is where we populate the `image_tags` variable that terraform uses to identify overridden tag values. I tried making this an HCL value, but TF wasn't happy with it. It's just a json-encoded string for now 😢